src/runtime/types.ts

```diff
@@ -16,3 +19,11 @@ declare module "h3" {
     $fetch: typeof globalThis.fetch;
   }
 }
+
+export interface RuntimeNitroHooks {
+  "render:response": (
```

Comment:
There's "render:html" too right ?

Reply:
Quite right! But that is Nuxt-specific and so Nuxt would augment this interface with that hook, as well as the island render hook.

https://github.com/nuxt/nuxt/blob/71ef8bd3ff207fd51c2ca18d5a8c7140476780c7/packages/nuxt/src/core/runtime/nitro/renderer.ts#L228